docs/user-guide/petset.md

```diff
@@ -143,16 +145,52 @@ Lets verify this assertion with a simple test.
 $ kubectl get svc
 NAME          CLUSTER-IP     EXTERNAL-IP       PORT(S)   AGE
 nginx         None           <none>            80/TCP    12m
+...
+```
+
+First, note that the PetSet gives us a stable hostname:
+
+```shell
+$ for i in 0 1; do kubectl exec web-$i -- sh -c 'hostname'; done
+web-0
+web-1
+```
+
+And the hostname is linked to the in-cluster DNS address:
+
+```shell
+$ kubectl run -it --image busybox dns-test --restart=Never /bin/sh
+dns-test # nslookup web-0.nginx
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+
+Name:      web-0.nginx
+Address 1: 10.180.3.5
+
+dns-test # nslookup web-1.nginx
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+
+Name:      web-1.nginx
+Address 1: 10.180.0.9
 ```
 
-The containers are running nginx webservers, which by default will look for an index.html file in `/usr/share/nginx/html/index.html`. That directory is backed by a PersistentVolume created by the Pet Set. So lets write our hostname there (remember the Pet Set gives us a stable hostname):
+The containers are running nginx webservers, which by default will look for an index.html file in `/usr/share/nginx/html/index.html`. That directory is backed by a PersistentVolume created by the Pet Set. So lets write our hostname there:
```

Comment:
That directory is backed by a `PersistentVolume` created by the Pet Set. So let's write our hostname there: